src/decode.c

```diff
@@ -656,7 +656,8 @@ inline int PacketSetData(Packet *p, const uint8_t *pktdata, uint32_t pktlen)
     if (unlikely(!pktdata)) {
         return -1;
     }
-    p->ext_pkt = (uint8_t *)pktdata;
```

Comment:
what does 'check' in the commit message refer to?

Reply:
I meant some keyword for "adding compiler check" (which is what `const` keyword does)
How would you name it ?